cfe_internal/update/update_policy.cf

```diff
@@ -390,6 +398,7 @@ bundle agent cfe_internal_update_cmdb_data_distribution
       # Write out the data for each host that had a data change
       "$(sys.workdir)/cmdb/$(_i)/host_specific.json"
         create => "true", # CFE-2329, ENT-4792
+        acl => u_acl_cmdb_host_specific_json, # ENT-11809
         template_data => mergedata("_get_cmdb_data_response_d[data][$(_i)]" ), # mergedata() is necessary in order to pick out a substructure, parsejson() is insufficient because expanding a key results in iteration of /values/ under that key
         template_method => "inline_mustache",
```

Comment:
I hope/believe we no longer use this promise to **render** these files. Is this the only place that manages permissions of these files?

Reply:
As far as I can tell, yes :)

Reply:
I believe it's definitely time to drop it, I believe `cf-reactor` has proven itself good in keeping those files up to date and this policy doesn't take group CMDB into account so it couldn't be used anyway. However, we should have a promise for correct permissions on these files.

Reply:
The bundle itself is guarded with !have_cf_reactor|cmdb_data_files_updates_done_in_policy.

```
    policy_server.enterprise_edition.(!have_cf_reactor|cmdb_data_files_updates_done_in_policy)::
      "cfe_internal_update_cmdb_data_distribution";
```

So, cf-reactor is taking care of this. The policy needs to remain until there are no supported versions who don't use cf-reactor for this.

Reply:
That also means that the acl on this file will not be managed, since cf-reactor is taking care of things.

Reply:
Thanks for the explanation 🚀